apps/vmq_server/src/vmq_cluster.erl

```diff
@@ -167,7 +167,10 @@ check_ready([Node|Rest], Acc) ->
                        _ -> false
                    end,
     ok = vmq_cluster_node_sup:ensure_cluster_node(Node),
```

Comment:
we could possibly get rid of the rpc call alltogether, no?

Reply:
The netsplit tests break if I remove the rpc call ... I'll have to figure out which assumption I'm breaking there.

Reply:
This was because removing the RPC call broke the netsplit tests as they only disrupt the erlang distribution (by changing the cookie) and don't do anything regarding the erlang cluster communications channel. For now I changed the netsplit code to do a `vmq_cluster_node_sup:del_cluster_node/1` and `vmq_cluster_node_sup:ensure_cluster_node/1` as well. I have a feeling this is not the right way as these calls might have side-effects which shouldn't happen when simulating a netsplit.

Reply:
thinking a bit more about it, removing the RPC calls defeats on purpose: namely that I'd like the check to check that both erlang distribution and the cluster comm are both working to declare the cluster ok. The question is, can we replace the rpc call with something less expensive?